x-pack/solutions/security/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.test.tsx

```diff
@@ -111,7 +111,7 @@ describe('<ComplianceDashboard />', () => {
           cspm: { status: 'not-installed', healthyAgents: 0, installedPackagePolicies: 0 },
           isPluginInitialized: false,
           indicesDetails: [
-            { index: 'logs-cloud_security_posture.findings_latest-default', status: 'empty' },
+            { index: 'logs-cloud_security_posture.misconfiguration_latest', status: 'empty' },
```

Comment:
`security_solution-cloud_security_posture.misconfiguration_latest` and in other places in this file